client/app/visualizations/chart/chart-editor.html

```diff
@@ -159,9 +159,9 @@
     <div class="form-group">
       <label class="control-label">Scale</label>
       <ui-select ng-model="options.xAxis.type">
-        <ui-select-match placeholder="Choose Scale...">{{$select.selected | capitalize}}</ui-select-match>
-        <ui-select-choices repeat="scaleType in xAxisScales">
-          <div ng-bind-html="scaleType | capitalize | highlight: $select.search"></div>
+        <ui-select-match placeholder="Choose Scale...">{{$select.selected.label | capitalize}}</ui-select-match>
```

Comment:
The `label` field is already supposed to be human friendly string, why not properly captialize instead of relaying on this filter?